tests/e2escenarios/e2e_source_test.go

```diff
@@ -1,33 +1,29 @@
 package e2escenarios
 
 import (
-	"os"
 	"path/filepath"
 	"runtime"
-	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/openshift/odo/tests/helper"
 )
 
 var _ = Describe("odo source e2e tests", func() {
-	var project string
-	var context string
 	var oc helper.OcRunner
```

Comment:
can we remove oc runnner also? we are getting cli runner in `commonVar`

Reply:
> can we remove oc runnner also? we are getting cli runner in commonVar

@adisky We can not replace `OcRunner` with `clirunner` specially in s2i because most of the functions in oc helper library are not common to kubectl and hence not there in `cliRunner` interface. However we are using such functions in s2i test scripts.

Reply:
what i get from this function https://github.com/openshift/odo/blob/master/tests/helper/helper_generic.go#L251 `clirunner` is returning `OcRunner`  for openshift clusters so its the same?

Reply:
discused bjns got it, Thanks